CSharp/README.md

```diff
@@ -314,6 +315,84 @@ public async Task<int> GetLatestPositionAsync()
 }
 ```
 
+## Testing Code
+
+We all know testing code is not built the same way that regular code is, because its purpose is different. This is why we have special guidelines for this type of code.
+
+### Test Class naming: {TargetTestClass}Tests
+
+```csharp
+// Bad
+namespace MyProject.UnitTests
+{
+    [TestClass]
+    public class MyService
+    { }
+}
+
+// Good
+namespace MyProject.UnitTests

```

Comment:
Ha, that's a very good idea. I haven't thought of it before.

Also, thanks for those engineering guidelines, I didn't know they existed.

I'll make the adjustment in this section.